tests/test_perf_tracking.py

```diff
@@ -2189,12 +2189,10 @@ class TestPositionTracker(unittest.TestCase):
     def setUpClass(cls):
         cls.env = TradingEnvironment()
 
-        equities_metadata = {1: {'asset_type': 'equity'},
-                             2: {'asset_type': 'equity'}}
-        futures_metadata = {3: {'asset_type': 'future',
-                                'contract_multiplier': 1000},
-                            4: {'asset_type': 'future',
-                                'contract_multiplier': 1000}}
+        equities_metadata = {1: {},
```

Comment:
This could be provided as identifiers to avoid the unnecessary empty dicts:
```
equities_identifiers = [1, 2]
cls.env.write_data(equities_identifiers=equities_identifiers,
                              futures_data=futures_metadata)
```